src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -257,6 +258,19 @@ int goto_instrument_parse_optionst::doit()
       return CPROVER_EXIT_SUCCESS;
     }
 
+    if(cmdline.isset("insert-final-assert-false"))
+    {
+      log.status() << "Inserting final assert false" << messaget::eom;
+      bool success = do_insert_final_assert_false(
+        goto_model,
+        cmdline.get_value("insert-final-assert-false"),
+        ui_message_handler);
+      if(!success)
```

Comment:
We should also return & exit on success, right? Suggest
`return fail ? CPROVER_EXIT_INTERNAL_ERROR : CPROVER_EXIT_SUCCESS;`

Reply:
We need to fall-through because we want to write out the modified goto program.